### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by kaishijeng** [Hexmagic/LaneDetection#2]
From train.py, it seems loss function is sum of dice loss and focus loss.
Is this true?
Do you have tried training with dice loss or focus loss only?

Thanks,

**Comment by Hexmagic** [comment]
single focal loss not worked well

**Comment by kaishijeng** [comment]
Thanks for an answer.
The trained model (myself or yours) doesn't perform very well on my own videos. My trained model resolution is at stage 2. Not able to train with stage 3 resolution due to GPU memory.
Any idea to improve this?

Thanks,

**Comment by Hexmagic** [comment]
1. use free colab  get more gpu memory
2. simplfy model architecture or use other model ,I suggest you have a look at https://github.com/qubvel/segmentation_models.pytorch
3. change image preprocessing
4. 16 bit training ,reference to https://github.com/PyTorchLightning/pytorch-lightning,[this worked on some gpu ]
